hooks/open-telemetry/pkg/metrics.go

```diff
@@ -149,6 +149,14 @@ func WithFlagMetadataDimensions(descriptions ...DimensionDescription) MetricOpti
 	}
 }
 
+// WithAttributeSetterForMetrics allows to set a extractionCallback which accept openfeature.FlagMetadata and returns
+// []attribute.KeyValue derived from that metadata.
+func WithAttributeSetterForMetrics(callback *func(openfeature.FlagMetadata) []attribute.KeyValue) MetricOptions {
```

Comment:
Maybe `WithTracesAttributeSetter` or `WithTracesAttributeMapper`.

Reply:
Renamed to `WithMetricsAttributeSetter`